.github/.jira_sync_config.yaml

```diff
@@ -0,0 +1,20 @@
+settings:
+  jira_project_key: "OPENG"
+  status_mapping:
+    opened: Untriaged
+    closed: done
+    not_planned: rejected
+    
+  components:
+    - loki
```

Comment:
I wonder if it shouldn't be `loki-coordinator`.

Reply:
After brief discussion, we're going with just `loki` for both :)